tests/chainer_tests/functions_tests/activation_tests/test_crelu.py

```diff
@@ -0,0 +1,131 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import functions
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+class _TestCReLU(unittest.TestCase):
```

Comment:
Use parameterized test. I mean:
```
@testing.parameterize(
  {'shape': (5, 4), 'axis': 0}, 
  {'shape': (5, 4), 'axis': 1}, 
  ....
)
```
You can access these values as an attribute of the test class like `self.shape`.